### PR TITLE
PPC RZIL tests

### PR DESCRIPTION
**Pull request by Rot127**
This PR adds the test binaries for the PPC uplifting.
It adds:
- README with documentation
- A build script
- A script to run all rz-tracetests
- Four binaries
  - `asm_tests`: Holds all implemented instructions. Disassembly is meant to be copied to `rizin/test/db/asm/ppc64`
  - `pseudo_fuzz_tests`: Copy of `asm_tests` but is compiled to a ELF file and run with QEMU. Each instruction class is given static random data for testing.
  - `ppc64le`, `ppc32be` Big and little endian binaries which are meant to test edge cases as well as branch instructions. Should more or less cover all possible execution paths in the RZIL code.

The builds of the ELF binaries are not reproducible. If this is a problem?

### CODE REVIEW

**Comment by Rot127** [comment]
Please do not merge. The binary is currently 17M large. Which is really not necessary.

File: elf/ppc/ppc64_fp.S
Comment (wargio) [inline elf/ppc/ppc64_fp.S:114]:
```suggestion
    # 64bit compiler (musl and gnu) doesn't support these instruction
```

**Review by XVilka — APPROVED** [review]

**Comment by XVilka** [comment]
@Rot127 GitHub says there is some conflict - please rebase and merge.

**Comment by wargio** [comment]
not to me., but yeah, rebase just in case

**Comment by Rot127** [comment]
Probably happened because I pushed to the same branch after it was merged before.

**Comment by Rot127** [comment]
Those two binaries the test fails for should be removed.

**Comment by thestr4ng3r** [comment]
> Those two binaries the test fails for should be removed.

Why? We have the policy to not allow any removals to prevent breaking tests of older rizin commits.

**Comment by Rot127** [comment]
> Why? We have the policy to not allow any removals to prevent breaking tests of older rizin commits.

I added these binaries for testing in the beginning of https://github.com/rizinorg/rizin/pull/2823 but quickly stopped using them. Those two were merged in https://github.com/rizinorg/rizin-testbins/pull/73 and https://github.com/rizinorg/rizin-testbins/pull/72 before https://github.com/rizinorg/rizin/pull/2823 was ready for review. The current PPC RZIL tests don't use them anymore.

If https://github.com/rizinorg/rizin/pull/2823 is squashed there will be no commit which uses them.

**Review by thestr4ng3r — APPROVED** [review]
I see, then lgtm.

**Pull request merged**
